osidb/models.py

```diff
@@ -80,29 +85,12 @@ def search_helper(
     # Order remaining results from highest rank to lowest
 
 
-class FlawHistoryManager(models.Manager):
+class FlawHistoryManager(ACLMixinManager):
     """flaw history manager"""
 
     def get_queryset(self):
```

Comment:
I wonder whether this is even necessary or it would be just enough to inherit it

Reply:
Agreed

Reply:
I believe I'm missing something here but I think the option of inherit only the `ACLMixin` does not work, let me take the `Flaw` model for example:
```
class ACLMixinManager(models.Manager):
    def get_queryset(self):
    ....
    
class ACLMixin(models.Model):
    objects = ACLMixinManager()
    ....
    
class FlawManager(models.Manager):
     def create_flaw(...):
     def fts_search(...):
     ...
     
class Flaw(... , ACLMixin):
    objects = FlawManager()
```

The `objects = FlawManager()` will replace the ACLMixin manager completely, poiting it to FlawManager's  `get_queryset` method (or to default if not set) and the models are using specific managers to override other methods besides `get_queryset`, making them necessary. 

Is there other solution available here?

Reply:
no no, I think what @osoukup meant is that we don't need to override the `get_queryset()` method since all we do is call `super`, so let it be inherited from `ACLManagerMixin`

Reply:
Exactly :smiley_cat:

Reply:
oh now I see. Removed the method not in use.